test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -42,7 +42,7 @@ e.emit('hello', 'a', 'b');
 
 
 // just make sure that this doesn't throw:
-var f = new events.EventEmitter();
+var f = new EventEmitter();
```

Comment:
Switched to `const` and did some refactoring of this file for scoping, dropping the on exit handler, preferring `strictEqual()`, etc.